src/completion-listener.js

```diff
@@ -2,9 +2,10 @@ const Rx = require('rxjs');
 const { bufferCount, switchMap, take } = require('rxjs/operators');
 
 module.exports = class CompletionListener {
-    constructor({ successCondition, scheduler }) {
+    constructor({ successCondition, scheduler, showTimings}) {
```

Comment:
Why was this needed?

Reply:
its a leftover from an old idea, will remove